apis/cache/v1beta1/referencers.go

```diff
@@ -16,37 +16,47 @@ limitations under the License.
 
 package v1beta1
 
-// // ResolveReferences of this ReplicationGroup
-// func (mg *ReplicationGroup) ResolveReferences(ctx context.Context, c client.Reader) error {
-// 	r := reference.NewAPIResolver(c, mg)
-
-// 	// Resolve spec.forProvider.securityGroupIDs
-// 	mrsp, err := r.ResolveMultiple(ctx, reference.MultiResolutionRequest{
-// 		CurrentValues: mg.Spec.ForProvider.SecurityGroupIDs,
-// 		References:    mg.Spec.ForProvider.SecurityGroupIDRefs,
-// 		Selector:      mg.Spec.ForProvider.SecurityGroupIDSelector,
-// 		To:            reference.To{Managed: &v1beta1.SecurityGroup{}, List: &v1beta1.SecurityGroupList{}},
-// 		Extract:       reference.ExternalName(),
-// 	})
-// 	if err != nil {
-// 		return err
-// 	}
-// 	mg.Spec.ForProvider.SecurityGroupIDs = mrsp.ResolvedValues
-// 	mg.Spec.ForProvider.SecurityGroupIDRefs = mrsp.ResolvedReferences
-
-// 	// Resolve spec.forProvider.cacheSecurityGroupNames
-// 	mrsp, err = r.ResolveMultiple(ctx, reference.MultiResolutionRequest{
-// 		CurrentValues: mg.Spec.ForProvider.CacheSecurityGroupNames,
-// 		References:    mg.Spec.ForProvider.CacheSecurityGroupNameRefs,
-// 		Selector:      mg.Spec.ForProvider.CacheSecurityGroupNameSelector,
-// 		To:            reference.To{Managed: &v1beta1.SecurityGroup{}, List: &v1beta1.SecurityGroupList{}},
-// 		Extract:       v1beta1.SecurityGroupName(),
-// 	})
-// 	if err != nil {
-// 		return err
-// 	}
-// 	mg.Spec.ForProvider.CacheSecurityGroupNames = mrsp.ResolvedValues
-// 	mg.Spec.ForProvider.CacheSecurityGroupNameRefs = mrsp.ResolvedReferences
-
-// 	return nil
-// }
+import (
```

Comment:
This change seems tangential. Should it be in its own commit?

Reply:
oops I actually already merged a commit to fix that, I'll rebase off master

Reply:
@negz okay should be ready now :+1: I think CI is acting up though